install/kubernetes/quick-install.yaml

```diff
@@ -121,6 +121,7 @@ data:
 
   install-iptables-rules: "true"
   auto-direct-node-routes: "false"
+  kube-proxy-replacement:  "probe"
```

Comment:
Shouldn't all other options be enabled by default in helm? `external-ips` `node-port`, `host-rechable-services`

Reply:
I don't have a strong opinion on this. If we default `kube-proxy-replacement` to `probe`, then majority of users won't need to mess with those options anyway. If I user sets the option to `strict` or `partial`, then we assume that they know what they are doing, so they can set manually each individual option.